9.4/root/usr/bin/run-postgresql

```diff
@@ -2,8 +2,9 @@
 
 set -eu
 
-source ${HOME}/common.sh
+source "$CONTAINER_SCRIPTS_PATH"/common.sh
```

Comment:
If I read correctly, its about *quoting* strings containing *variables*, which I actually done because the original code suffered from it.  I'll update according to your style .. a sec please.